### ISSUE TITLE
IOS support

### ISSUE THREAD

**Issue by rekonvald** [ConnectyCube/connectycube-flutter-call-kit#3]
Hi!

Your package is exactly what I need! But we need ios support also. 

Thank you

**Comment by TatankaConCube** [comment]
We plan to do this in a few next releases))

**Comment by rlee1990** [comment]
@TatankaConCube are there any updates on this? Also does this work as a stand-alone?

**Comment by TatankaConCube** [comment]
unfortunately no(((  we still use the separated plugin for iOS:
```yaml
flutter_call_kit:
    git:
      url: git://github.com/raj457036/flutter_call_kit.git
      ref: master
```

**Comment by TatankaConCube** [comment]
happy to inform you guys that today we released the new version [2.0.0](https://pub.dev/packages/connectycube_flutter_call_kit/versions/2.0.0) with iOS support, enjoy it
